docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -955,6 +955,10 @@ Frequency of renewing this node's average uptime metric. Defaults to `30s`.
 
 ##### `--snow-avalanche-batch-size` (int)
 
+:::caution
```

Comment:
This was not deprecated - it was fully removed

Reply:
true, my bad. fixed 1fa6ed428ac455d0293f6cd97289fd50fb6764e2